docs/docusaurus/docs/guides/expectations/advanced/how_to_create_a_new_expectation_suite_using_rule_based_profilers.md

```diff
@@ -38,7 +38,7 @@ While the contents of this document accurately reflect the state of the feature,
 
 ### 3. Create a context and add your Datasource
 
-- Follow the steps in the [How to connect to data on a filesystem using Pandas](../../../guides/connecting_to_your_data/filesystem/pandas.md). The following code snippet adds a Pandas Filesystem asset for our taxi data.
+- Follow the steps in the [How to connect to data on a filesystem using Pandas](/docs/0.15.50/guides/connecting_to_your_data/filesystem/pandas). The following code snippet adds a Pandas Filesystem asset for our taxi data.
```

Comment:
```suggestion
- See [How to connect to data on a filesystem using Pandas](/docs/0.15.50/guides/connecting_to_your_data/filesystem/pandas). Run the following command to add a Pandas Filesystem asset for the taxi data.
```

It appears that pandas is not capitalized:

![image](https://github.com/great-expectations/great_expectations/assets/104205257/941a0a7e-3fcf-4a55-a26c-1b0dbe1539fe)